release template/SECURITY.md

```diff
@@ -28,4 +28,4 @@ This information will help us triage your report more quickly.
 
 ## Policy
 
-See [GitHub's Safe Harbor Policy](https://docs.github.com/en/github/site-policy/github-bug-bounty-program-legal-safe-harbor#1-safe-harbor-terms)
+See [<COMPANY_NAME>'s Safe Harbor Policy](XXX)
```

Comment:
That seems to be another placeholder document that might be needed

Reply:
I'm thinking I'll leave this one since it only appears once. Users should still be able to find it when they search for `XXX` which we still instruct them to do in the `README`. Hopefully that makes sense.